packages/pieces/telegram-bot/src/lib/action/send-text-message.action.ts

```diff
@@ -28,23 +31,47 @@ export const telegramSendMessageAction = createAction({
         }),
         message_thread_id: Property.ShortText({
             displayName: 'Message Thread Id',
-            description: 'Unique identifer for the target message thread of the forums; for forums supergroups only',
+            description: 'Unique identifier for the target message thread of the forums; for forums supergroups only',
             required: false,
         }),
+        format: Property.StaticDropdown({
+            displayName: 'Format',
+            description: 'Choose format you want ',
+            required: true,
+            options: {
+                options: [
+                    {
+                        label: 'Markdown',
+                        value: 'MarkdownV2'
+                    },
+                    {
+                        label: 'HTML',
+                        value: 'HTML'
+                    }
+                ]
+            },
+            defaultValue: 'MarkdownV2'
+
+        }),
+        instructions_format: Property.MarkDown({
+            value: format,
+        }),
         message: Property.LongText({
             displayName: 'Message',
             description: 'The message to be sent',
             required: true,
         })
     },
     async run(ctx) {
+        console.log(ctx)
         return await httpClient.sendRequest<never>({
             method: HttpMethod.POST,
             url: telegramCommons.getApiUrl(ctx.auth, 'sendMessage'),
             body: {
                 chat_id: ctx.propsValue['chat_id'],
                 text: ctx.propsValue['message'],
                 message_thread_id: ctx.propsValue['message_thread_id'] ?? undefined,
+                parse_mode: ctx.propsValue['format']
```

Comment:
Please add `ctx.propsValue['format'] ?? 'MarkdownV2' ` as the field is no longer required